docs/src/content/docs/developer-guides/payments/index.mdx

```diff
@@ -0,0 +1,185 @@
+
+const walletBalance = await synapse.payments.walletBalance(TOKENS.USDFC);
+const accountInfo = await synapse.payments.accountInfo();
+
+console.log("Available:", accountInfo.availableFunds);
+console.log("Locked:", accountInfo.lockupCurrent);
+console.log("Spending rate:", accountInfo.lockupRate, "per epoch");
+```
+
+### Fund Your Account
+
+```ts twoslash
+import { Synapse } from "@filoz/synapse-sdk";
+// @ts-ignore
+const synapse = await Synapse.create();
+// ---cut---
+import { ethers } from "ethers";
+import { TOKENS } from "@filoz/synapse-sdk";
+
+const amount = ethers.parseUnits("100", 18);
+const tx = await synapse.payments.depositWithPermit(amount, TOKENS.USDFC);
+await tx.wait();
+```
+
+<details>
+<summary>*Alternative: Two-Transaction Deposit (not recommended)*</summary>
+
+If you cannot use permit-based deposits, you can use the traditional two-transaction approach:
+
+```ts
+// 1. Approve USDFC spending
+await usdfc.approve(paymentAddress, depositAmount);
+
+// 2. Deposit
+await synapse.payments.deposit(depositAmount);
+```
+
+This requires two transactions and higher gas costs. Use `depositWithPermit` instead.
+
+</details>
+
+### Account Health Monitoring
+
+**Important**: Monitor your account health regularly. Insufficient balance causes payment failures and service interruptions.
+
+```ts twoslash
+import { Synapse } from "@filoz/synapse-sdk";
+// @ts-ignore
+const synapse = await Synapse.create();
+// ---cut---
+import { TIME_CONSTANTS } from "@filoz/synapse-sdk";
+
+const info = await synapse.payments.accountInfo();
+const epochsRemaining = info.availableFunds / info.lockupRate;
+const daysRemaining =
+  Number(epochsRemaining) / Number(TIME_CONSTANTS.EPOCHS_PER_DAY);
+
+console.log(`Days remaining: ${daysRemaining.toFixed(1)}`);
+if (daysRemaining < 7) console.warn("⚠️ Low balance!");
+```
+
+### Withdrawing Unlocked Funds
+
+```ts twoslash
+import { Synapse } from "@filoz/synapse-sdk";
+// @ts-ignore
+const synapse = await Synapse.create();
+// ---cut---
+const info = await synapse.payments.accountInfo();
+await synapse.payments.withdraw(info.availableFunds);
+```
+
+## Approving Operators
+
+Operators are smart contracts authorized to spend your tokens for specific services. Before uploading files, you must approve the WarmStorage operator. This approval is required only once per operator and grants permission to create payment rails on your behalf.
+
+Three types of allowances protect your funds from unauthorized spending:
+
+1. **Rate Allowance** - Max spending per epoch across all rails
+2. **Lockup Allowance** - Max total funds locked across all rails
+3. **Max Lockup Period** - Max duration funds can be locked (the safety net)
+
+### Approving an Operator
+
+Approving warmStorage for 1TiB of storage for 3 months.
+
+```ts twoslash
+import { Synapse } from "@filoz/synapse-sdk";
+// @ts-ignore
+const synapse = await Synapse.create();
+const monthlyPricePerTiB = null as unknown as bigint;
```

Comment:
What does this mean?  Can we make as a thing before code is offered for review it works; this seems odd to me.

Reply:
I'm still a little bit unclear as to keeping `const monthlyPricePerTiB = null as unknown as bigint;` in the code sample?